pallets/gear/src/tests.rs

```diff
@@ -905,7 +906,7 @@ fn send_reply_failure_to_claim_from_mailbox() {
             panic!("Program is terminated!");
         };
 
-        populate_mailbox_from_program(prog_id, USER_1, 2, 20_000_000, 0);
```

Comment:
Test somehow failing with the old value

Reply:
Why?

Reply:
I don't understand actually. I don't see any reasons to fail